packages/playground/website/src/components/site-manager/site-info-panel/index.tsx

```diff
@@ -38,10 +38,15 @@ function SiteInfoRow({
 	label: string;
 	value: string | JSX.Element;
 }) {
+	const labelClassSufix = label.toLowerCase().replace(/\s+/g, '-');
 	return (
 		<Flex justify="flex-start" expanded={true}>
 			<FlexItem className={css.infoRowLabel}>{label}</FlexItem>
-			<FlexItem className={css.infoRowValue}>{value}</FlexItem>
+			<FlexItem
```

Comment:
I avoided adding code only for testing, but there weren't any good selectors that I could come up with for this element so I added a class.

Reply:
What are Playwright's recommendations in cases like that? I know React Testing Library recommends semantic role or aria attributes instead of classes.

Reply:
Before creating dedicated selectors, you can try letting Playwright pick the selector for you via the Playwright Inspector. One of the ways to run your app with Inspector open is the following:

```
npx playwright open http://localhost:3000` # or whatever the URL you're testing
```

For further info, see https://playwright.dev/python/docs/debug#picking-locators

Reply:
> What are Playwright's recommendations in cases like that? I know React Testing Library recommends semantic role or aria attributes instead of classes.

Playwright also [recommends using user-facing attributes](https://playwright.dev/docs/best-practices#prefer-user-facing-attributes-to-xpath-or-css-selectors)!

Reply:
Thanks for the feedback.

I added _Playwright Inspector_ to the README.

> Playwright also [recommends using user-facing attributes](https://playwright.dev/docs/best-practices#prefer-user-facing-attributes-to-xpath-or-css-selectors)!

This helps a lot, I ended up switching to aria-label. It makes more sense here.
We could use `getByText` in some cases, but in some like networking it wouldn't work well (`getByText('No')`).